pkg/broker/broker.go

```diff
@@ -42,6 +45,9 @@ type Client struct {
 	// Each connection has an id that corresponds to the Event ID it is
 	// interested in receiving messages about.
 	ID string
+
+	// Extra contains any extra misc information about the connection.
```

Comment:
Like what? Can we provide an example?

Reply:
`Extra` may include information such as the path of a file to be read. Added an example.